keps/sig-node/2400-node-swap/kep.yaml

```
@@ -33,7 +33,7 @@ latest-milestone: "v1.30"
 milestone:
   alpha: "v1.22"
   beta: "v1.30"
-  stable: "TBD"
+  stable: "v1.32"
```

```suggestion
  #stable: "v1.32"
```
(AIUI)
We can put in a suggestion, but until we're planning that release, it's not a commitment. If we are planning that release, we should change `latest-milestone` and `stage` I think.

---

I think that we should indeed plan it for next release.

The `latest-milestone` comments read:
```
# The most recent milestone for which work toward delivery of this KEP has been
# done. This can be the current (upcoming) milestone, if it is being actively
# worked on.
```

The milestone's documentation reads:
```
# The milestone at which this feature was, or is targeted to be, at each stage.
```

Since we target 1.32, `miltstone` seemed like the right place. I've added it now to `latest-milestone`.

What's important to me is that we indeed plan and commmit to releasing in 1.32, which there's a broad consensus of doing. I've even volunteered to not pursue GAing in 1.31 to reduce pressure from the sig-node approvers and ensure we do so in 1.32 with no pressure.